### ISSUE TITLE
NVDA reports "column 0 of 1" in MS Word

### ISSUE THREAD

**Issue by feerrenrut** [nvaccess/nvda#6995]
When using NVDA (version `next-13911,3c8e30bd`) with Microsoft Word 2013 using [this document](https://github.com/nvaccess/nvda/files/857290/col.zero.of.one.zip). Move by line onto page 2 and "column 0 of 1" is reported.

**Comment by surfer0627** [comment]
I have a similar case, which  occurs in NVDA_2017.1.

1. Open the [attachment](https://github.com/nvaccess/nvda/files/857347/report_text_columns.docx).
2. Press ctrl+alt+arrows to move to row3 column2.
3. Move to row3 column1.
NVDA report: column1 column 0 of 1
4. Move to row3 column2.
NVDA report: column2 column 1 of 1

**Comment by surfer0627** [comment]
If user disabled "Report page numbers" option in the Document Formatting dialog, this case willl not occur.

**Comment by Adriani90** [comment]
@surfer0627, @feerrenrut I am testing with NVDA 2019.1 and Office 2010 as well as Office 2016 / 365. I hahve also tried both with UIA enabled and disabled. i cannot reproduce your issues regarding column report. Are you still having this problem?

**Comment by Adriani90** [comment]
We didn't get any update since many years on this issue, I am closing as abandoned given also that support for Office 2010 ended completely while support for Office 2013 will completely end in April 2023.

If you are still having this issue in office 2016 or newer, please comment on this and we can reopen.
